rbac/server/api/users.py

```diff
@@ -185,6 +185,8 @@
         request.app.config.VAL_CONN, batch_list, request.app.config.TIMEOUT
     )
 
+    await reject_users_proposals(next_id, request)
```

Comment:
We can mimic LDAP delta sync by having reject_users_proposals create the transactions and appending those transactions in the txn_list that gets instantiated in line 173.